### PR TITLE
Updated 2 presences and releas statcord presence

### PR DESCRIPTION
**Pull request by ModBotStudio**
Discordlabs: Added support for 2 more sites.
Usercord: Added support for api site.
Statcord: Made new presence.
image
image
image

### CODE REVIEW

File: statcord.com/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    "description": {
+      "en": "Statcord makes it easy to grow your Discord Bot. Connect your bot and get really nice statistics on whats going on.",
+      "pl": "Statcord ułatwia rozwój Discord Botów. Połącz swojego bota i uzyskaj naprawdę ładne statystyki na temat tego, co się dzieje."
+    },
+    "url": [
+      "statcord.com",
+      "docs.statcord.com"
+    ],
+    "version": "1.0.0",
+    "logo": "https://cdn.statcord.com/logo.png",
```
Comment (Bas950) [inline statcord.com/dist/metadata.json]: upload logo to imgur

File: statcord.com/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    "thumbnail": "https://i.imgur.com/2iJVEDe.png",
+    "color": "#6a0dad",
+    "tags": [
+      "Bots",
+      "Discord",
+      "Stats"
+    ],
```
Comment (Bas950) [inline statcord.com/dist/metadata.json]: tags must be lower case

File: Usercord.org/dist/metadata.json [hunk 3]
```diff
@@ -1,21 +1,24 @@
...
+    "docs.usercord.org"
+  ],
+  "version": "1.1.3",
+  "logo": "https://i.imgur.com/cVIFD4l.png",
+  "thumbnail": "https://i.imgur.com/K7XK8SV.png",
+  "color": "#00ff00",
+  "tags": [
+    "Users",
+    "Discord",
+    "List"
```
Comment (Bas950) [inline Usercord.org/dist/metadata.json]: tags must be lower case

**Comment by frozaum** [comment]
also your fork is outdated, so yea

**Comment by ModBotStudio** [comment]
> DiscordLabs.org/dist/presence.js:
> 
> Duplicated else if statement, also submit is spelt wrong.
> 

There are no Duplicated

File: statcord.com/dist/metadata.json [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    "service": "Statcord.com",
+    "description": {
+      "en": "Statcord makes it easy to grow your Discord Bot. Connect your bot and get really nice statistics on whats going on.",
+      "pl": "Statcord ułatwia rozwój Discord Botów. Połącz swojego bota i uzyskaj naprawdę ładne statystyki na temat tego, co się dzieje."
+    },
+    "url": [
+      "statcord.com",
+      "docs.statcord.com"
+    ],
+    "version": "1.0.1",
```
Comment (Bas950) [inline statcord.com/dist/metadata.json]: V1.0 on release or V1.0.0

File: DiscordLabs.org/dist/presence.js [hunk 5]
```diff
@@ -10,53 +10,57 @@ var browsingStamp = Math.floor(Date.now() / 1000);
...
+            var priceEls = document.getElementsByClassName("uname");
+            for (var i = 0; i < priceEls.length; i++) {
+                var profilename = priceEls[i].innerText;
+                presenceData.details = "Viewing a profile:";
+                presenceData.state = profilename
+            }
+
+        } else if (document.location.pathname.includes("/submit")) {
+            presenceData.details = "Viewing Page:";
+            presenceData.state = "Bot Sumbit";
```
Comment (Cairo2k18) [inline DiscordLabs.org/dist/presence.js]: Still sumbit, should be submit - regardless of a spelling mistake on the website.

File: statcord.com/dist/metadata.json (hunk 4 above)
Comment (ModBotStudio) [reply statcord.com/dist/metadata.json]: Done

**Comment by doomlerd** [comment]
Hello @ModBotStudio, your PR is not merged yet because we're making some changes in favor of quality assurance which will require your PR to be re-made, make sure to fix the [current reported issues](https://deepscan.io/dashboard/#view=project&tid=6792&pid=8925&bid=113944&subview=pull-request&prid=300490) and resolve the conflicts, then we'll inform you what to do next ( to comply with the new changes ), I'll keep this on hold now until we finish.

For reference : #1248 ( rebase #1258 )

**Comment by doomlerd** [comment]
Sorry for the wait, now that the repo is unlocked you have to recreate the PR with these changes :
format your code with either spaces or tabs and using only 2 or 4 in tab width, and run the following compiler command ( in every presence folder ) to recompile your files : 
`tsc --noFallthroughCasesInSwitch --noUnusedLocals --noUnusedParameters -t "ES2020"`
then recreate the PR for the new checks to start.

**Comment by ModBotStudio** [comment]
> Sorry for the wait, now that the repo is unlocked you have to recreate the PR with these changes :
> format your code with either spaces or tabs and using only 2 or 4 in tab width, and run the following compiler command ( in every presence folder ) to recompile your files :
> `tsc --noFallthroughCasesInSwitch --noUnusedLocals --noUnusedParameters -t "ES2020"`
> then recreate the PR for the new checks to start.

I dont know how new code looks

**Pull request closed without merging**